### PR TITLE
Create validator profile for Terrarium

### PR DESCRIPTION
**Pull request by hanahorangi**

### CODE REVIEW

File: validators/terravaloper1k3202kcypl5csdtyvr3qjmxzzjnyn84ghj2r7y/profile.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "contact": {
+    "email": "admin@terrarium.community",
+    "twitter": "@ClanMudhorn_",
```
Comment (terran6) [inline validators/terravaloper1k3202kcypl5csdtyvr3qjmxzzjnyn84ghj2r7y/profile.json:4]: @hanahorangi Please remove the trailing comma to adhere to JSON syntax and then we can merge your profile.  Thanks!

**Review by terran6 — CHANGES_REQUESTED** [review]

**Review by hanahorangi — COMMENTED** [review]
Trailing comma removed

**Review by hanahorangi — COMMENTED** [review]
Trailing comma removed

**Comment by hanahorangi** [comment]
removed trailing comma

**Review by terran6 — APPROVED** [review]

**Review by terran6 — APPROVED** [review]

**Pull request merged**